python/py-plyvel/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-plyvel
+version             1.0.4
+categories-append   devel
+platforms           darwin
+license             BSD
+
+python.versions     27 34 35 36
+
+maintainers         {ipglider.org:miguel @ipglider} openmaintainer
+
+description         Plyvel is a fast and feature-rich Python interface to LevelDB.
+long_description    Plyvel is a Python interface to LevelDB that has a rich feature set, high performance and a friendly Pythonic API.
```

Comment:
This should probably be wrapped (with a `\` ) at 80 columns. That said, if I merge this before you fix it, submit another pull request to fix that later.